pkgs/by-name/in/incus-unwrapped/package.nix

```diff
@@ -77,9 +86,13 @@ buildGoModule rec {
     '';
 
   postInstall = ''
-    installShellCompletion --bash --name incus ./scripts/bash/incus
+    installShellCompletion --cmd incus \
+      --bash <($out/bin/incus completion bash) \
```

Comment:
You probably should use the separate bash completion for this one as ours knows how to complete instance names and a variety of other things.

Reply:
Hmm, maybe we can move that logic to cobra dynamic completion instead? Then all shells could benefit. https://github.com/spf13/cobra/blob/main/site/content/completions/_index.md#dynamic-completion-of-nouns

Reply:
Yeah, that's the longer term plan, been meaning to look into that for a few years now, if you have some time to spare, we'd definitely welcome the PR :)

Reply:
Reverted for bash until I can dig deeper into the cobra side.